backend/config/storage.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+type StorageConfig struct {
+	Path string `yaml:"path"`
+}
+
+func validateStorageConfig(cfg *Config) error {
+	if cfg.Storage == nil {
+		cfg.Storage = &StorageConfig{
+			Path: DefaultStoragePath,
+		}
+	}
+
+	if len(cfg.Storage.Path) == 0 {
+		cfg.Storage.Path = DefaultStoragePath
+	}
+
+	info, err := os.Stat(cfg.Storage.Path)
+	if os.IsNotExist(err) {
+		// if it is a local path, create it
+		if !strings.HasPrefix(cfg.Storage.Path, "/") &&
```

Comment:
These checks aren't comprehensive when checking if the path is local. a path like `./woo/../../bin` would succeed.

Reply:
I think I can write better checks with `path.Clean`